code/__DEFINES/multiz.dm

```diff
@@ -1,4 +1,8 @@
 /// Attempt to get the turf below the provided one according to Z traits
-#define GET_TURF_BELOW(turf) ((!(turf) || !length(SSmapping.multiz_levels) || !SSmapping.multiz_levels[(turf).z][Z_LEVEL_DOWN]) ? null : get_step((turf), DOWN))
+#define GET_TURF_BELOW(turf) ( \
+	(turf:turf_flags & RESERVATION_TURF) ? SSmapping.get_reservation_from_turf(turf)?.get_turf_below(turf) : \
```

Comment:
is : actually needed here? I'm fine with forcing people to properly type the arg to this macro